apps/gateway/src/worker.ts

```
@@ -231,14 +237,52 @@
+					error: r.error?.message,
+				})),
+			);
+		}
+
+		if (validMessages.length === 0) {
+			// All messages were invalid, remove them all
+			await removeFromProcessingQueue(LOG_PROCESSING_QUEUE, messages.length);
+			return;
+		}
+
+		const logData = validMessages.map((r) => r.data!);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'logData'. Consider to delete it or rename it.

'logData' is defined here:


(lint/suspicious/noRedeclare)